ProgramowanieBot/Handlers/MessageHandler.cs

```diff
@@ -1,16 +1,27 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 using NetCord;
 using NetCord.Gateway;
 
+using ProgramowanieBot.Data;
+using ProgramowanieBot.Helpers;
+
 namespace ProgramowanieBot.Handlers;
 
 internal partial class MessageHandler : BaseHandler<ConfigService.GuildThreadHandlerConfig>
 {
+    private readonly string[] _resolveKeywords;
+    private readonly double _maxReminders;
+    private readonly string _reminderMessage;
```

Comment:
I think we should get the values from config every time, `_typingTimeout` is an exception because we have to parse the value from config to get it, so it's more performant to cache it.

Reply:
Why, though? What's wrong with caching all the other config values?

Reply:
what's the point of caching them?

Reply:
I feel like this way it'd be easier to access them if they were used more than once. In this case, however, I think you're right. I'll commit the changes right away.